ui/lib/ldap/addon/components/page/overview.hbs

```diff
@@ -33,16 +33,15 @@
       </h2>
     </OverviewCard>
   </div>
-  <div class="is-flex-align-start has-top-margin-l">
+  <div class="is-grid has-top-margin-l grid-2-columns grid-gap-2">
     <AccountsCheckedOut
       @libraries={{@libraries}}
       @statuses={{@librariesStatus}}
       @showLibraryColumn={{true}}
       @onCheckInSuccess={{transition-to "vault.cluster.secrets.backend.ldap.overview"}}
       class="is-flex-half"
     />
-
-    <div class="has-left-margin-l is-flex-half">
+    <div>
```

Comment:
This wrapper div is necessary, otherwise the card height will expand to match the height of the other card when displayed in a row, leaving a lot of whitespace that does not match the designs.